site/gatsby-site/src/components/theme/themeProvider.js

```diff
@@ -37,11 +37,7 @@ class ThemeProvider extends React.Component {
     return (
       <div>
         <Global styles={baseStyles} />
-        <Header
-          location={location}
-          isDarkThemeActive={isDarkThemeActive}
-          toggleActiveTheme={this.toggleActiveTheme}
-        />
+        <Header isDarkThemeActive={isDarkThemeActive} toggleActiveTheme={this.toggleActiveTheme} />
```

Comment:
New Issue: remove dark theme support. I don't want to support it because the templating is going to be far more difficult to support as the feature set expands.